Abcrete/BangazonCLI
# Reviewed DatabaseInterface and fixed minor issues

## Status
Choose:
Ready

## Description

1. It will fix a a problem with not creating product table and all the tables in Caps
2. Wrote tests to add product, get product, and remove product

## Todos

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. git fetch
2. git checkout az-branch
3. delete database file in your file system 
4. try to run the tests and it should create all the tables 
5.  and all tests should pass

## Impacted Areas in Application
1. Fixed minor issues in DatabaseInterface
## Link to Feature Ticket
#33  #13
